### PR TITLE
[Mongo] Optimize `reports` collection query

### PR DESCRIPTION
**Pull request by mishig25**
As the title says, optimize `reports` collection query

### CODE REVIEW

File: src/routes/settings/+layout.server.ts [hunk 1]
```diff
@@ -21,10 +21,10 @@ export const load = (async ({ locals, parent }) => {
 				createdByMe:
 					el.createdById.toString() === (locals.user?._id ?? locals.sessionId).toString(),
 				reported:
-					(await collections.reports.countDocuments({
+					(await collections.reports.findOne({
```
Comment (mishig25) [inline src/routes/settings/+layout.server.ts:24]: this should be faster than aggregation of `countDocuments`

File: src/lib/server/database.ts [hunk 2]
```diff
@@ -85,5 +85,5 @@ client.on("open", () => {
 	assistants.createIndex({ userCount: 1 }).catch(console.error);
 	assistants.createIndex({ featured: 1, userCount: -1 }).catch(console.error);
 	assistants.createIndex({ modelId: 1, userCount: -1 }).catch(console.error);
-	reports.createIndex({ assistantId: 1 }).catch(console.error);
+	reports.createIndex({ createdBy: 1, assistantId: 1 }).catch(console.error);
```
Comment (coyotte508) [inline src/lib/server/database.ts:88]: I think it's nice to keep both indexes - that way we can easily see how many times a particular assistant has been reported, or which assistant is the most reported

Comment (mishig25) [reply src/lib/server/database.ts]: wdyt 2750cbcb322e04cd1d85f5fc969c6af486e7e976 ?

Comment (coyotte508) [reply src/lib/server/database.ts]: nice!

**Pull request merged**